#! /usr/bin/env python

PACKAGE='aricc_2d_vision'
try:
    import imp
    imp.find_module(PACKAGE)
    from dynamic_reconfigure.parameter_generator_catkin import *
except:
    import roslib; roslib.load_manifest(PACKAGE)
    from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

gen.add("left_top_x", double_t, 0, "the x of left_top", 1, -1000, 1000)
gen.add("left_top_y", double_t, 0, "the y of left_top", 1, -1000, 1000)
gen.add("width",      double_t, 0, "width",             1, 0, 1000)
gen.add("height",     double_t, 0, "height",            1, 0, 1000)
gen.add("z",          double_t, 0, "z",            0, 0, 1)
gen.add("project",    bool_t, 0,   "Project input", False)


exit(gen.generate(PACKAGE, "aricc_2d_vision", "RectToMaskImage"))
